plugin-tester-scala/build.gradle

```
@@ -5,55 +5,30 @@ buildscript {
   }
 
   dependencies {
-    // see https://plugins.gradle.org/plugin/com.lightbend.akka.grpc.gradle
-    // for the currently latest version.
-    //classpath 'gradle.plugin.com.lightbend.akka.grpc:akka-grpc-gradle-plugin:0.1'
-
-    // For testing on travis, we use another group id:
-    classpath 'com.lightbend.akka.grpc:akka-grpc-gradle-plugin:SNAPSHOT'
-
-    classpath "gradle.plugin.com.github.maiflai:gradle-scalatest:0.22"
+    classpath "com.lightbend.akka.grpc:akka-grpc-gradle-plugin:${project.getProperty("akkaGrpcTest.pluginVersion")}"
   }
 }
 
 plugins {
   id 'scala'
   id 'application'
-  id "com.github.maiflai.scalatest" version "0.22"
 }
 
 apply plugin: 'com.lightbend.akka.grpc.gradle'
 
-apply plugin: "com.github.maiflai.scalatest"
-
-// These are the default options for a Scala project (not necessary to define)
 akkaGrpc {
-  language = "Scala"
-  generateClient = true
-  generateServer = true
   serverPowerApis = true // This isn't default
 }
 
 repositories {
-  mavenLocal()
   mavenCentral()
 }
 
-dependencies {
-  testImplementation 'com.typesafe.akka:akka-stream-testkit_2.12:2.5.13'
-  testImplementation 'org.scalatest:scalatest_2.12:3.0.5'
-  testRuntime 'org.pegdown:pegdown:1.1.0'
+def scalaBase = "2.12"
```

Prefer `scala_major_version` (in some idiomatic format) instead of `scalaBase`.

```suggestion
def scalaMajorVersion = "2.12"
```

That's the preferred name and is used [widely](https://github.com/akka/akka-grpc-quickstart-scala.g8/blob/master/src/main/g8/default.properties). :-)

---

scalaBinaryVersion is also common

---

Pushed